LiteCore/Query/SQLiteFleeceFunctions.cc

```diff
@@ -103,6 +107,77 @@ namespace litecore {
         }
     }
 
+    static void handle_fts_value(const Value* data, alloc_slice& result) {
+        switch(data->type()) {
+            case kBoolean:
+                result.append(data->asBool() ? "T"_sl : "F"_sl);
```

Comment:
Why "T" and "F", not "true" and "false"? Compatibility with Server?

Reply:
Yes, that's where I got the values from